app/serializers/theme_settings_serializer.rb

```diff
@@ -6,7 +6,6 @@ class ThemeSettingsSerializer < ApplicationSerializer
              :default,
              :value,
              :description,
-             :objects_schema_property_descriptions,
```

Comment:
We were preloading the objects setting editor stuff as part of `ThemeSettingsSerializer` which is called by `ThemeSerializer`